### PR TITLE
1.16.0 beta.1 bump

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -34,15 +34,17 @@ import:
...
-  version: 89742aefa4b206dcf400792f3bd35b542998eb3b
+  version: v1.4.2
 - package: github.com/blang/semver
-  version: b38d23b8782a487059e8fc8773e9a5b228a77cb6
+  version: v3.5.0
 - package: github.com/imdario/mergo
-  version: 9316a62528ac99aaecb4e47eadd6dc8aa6533d58
+  version: v0.3.5
+- package: sigs.k8s.io/structured-merge-diff
+  version: 0c1d754dd648 # matches 1.16.0-beta.1 go.mod
```
Comment (mfojtik) [inline glide.yaml]: @sttts unit tests won't compile without this

File: glide.yaml [hunk 2]
```diff
@@ -53,7 +55,10 @@ import:
 - package: github.com/fsouza/go-dockerclient
   version: da3951ba2e9e02bc0e7642150b3e265aed7e1df3 # matching origin 4.2 level
 - package: github.com/docker/distribution
-  version: 16128bbac47f75050e82f7e91b04df33775e0c23 # level currently used in origin to base the origin patches on.  See https://github.com/openshift/image-registry/pull/126/commits/eb32acef7827ac2227c3aeaddc444880ed98edb3 leading to https://github.com/openshift/docker-distribution/commits/image-registry-3.11
+  # See https://github.com/openshift/image-registry/pull/126/commits/eb32acef7827ac2227c3aeaddc444880ed98edb3 leading to https://github.com/openshift/docker-distribution/commits/image-registry-3.11
+  version: 16128bbac47f75050e82f7e91b04df33775e0c23 # level currently used in origin to base the origin patches on.
+- package: github.com/docker/libnetwork
+  version: 7f13a5c99f4bb76a4122035d495984b6a09739bb # required for docker/docker/opts
```
Comment (mfojtik) [inline glide.yaml]: matches docker/docker level the image stuff need

Comment (mfojtik) [reply glide.yaml]: (unit test won't compile without this)

File: glide.yaml [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+  version: kubernetes-1.16.0-beta.1
 - package: k8s.io/kube-aggregator
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: k8s.io/client-go
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: github.com/openshift/api
-  version: master
+  version: prebase-1.16 # this is wrong, it should be prebase-1.16.0-beta.1
```
Comment (mfojtik) [inline glide.yaml]: @deads2k will fix us in afternoon

Comment (soltysh) [inline glide.yaml]: Details, once we're past prebase step we'll merge to master and it'll be already some .0 version ;)

**Pull request merged**